src/lib/math.js

```diff
@@ -18,8 +18,40 @@ const $builtinmodule = function (name) {
         return new Sk.builtin.float_(Math.ceil(_x));
     };
 
-    function comb(x, y) {
-        throw new Sk.builtin.NotImplementedError("math.comb() is not yet implemented in Skulpt");
+    function comb(n, k) {
+        n = Sk.misceval.asIndexOrThrow(n);
+        k = Sk.misceval.asIndexOrThrow(k);
+        if (n < 0) {
+            throw new Sk.builtin.ValueError("n must be an non-negative integer");
+        }
+        if (k < 0) {
+            throw new Sk.builtin.ValueError("k must be a non-negative integer");
+        }
+        if (k > n) {
+            return new Sk.builtin.int_(0);
+        }
+        n = new Sk.builtin.int_(n);
+        k = new Sk.builtin.int_(k);
+        let temp = n.nb$subtract(k);
+        if (Sk.ffi.remapToJs(temp) < Sk.ffi.remapToJs(k)) {
+            k = temp
+        }
+        if (Sk.ffi.remapToJs(k) === 0) {
+            return new Sk.builtin.int_(1);
+        }
+        if (Sk.ffi.remapToJs(k) > Number.MAX_SAFE_INTEGER) {
+            throw new Sk.builtin.OverflowError("k must not exceed " + Number.MAX_SAFE_INTEGER);
+        }
+
+        const one = new Sk.builtin.int_(1);
+        let tot = n;
+        for (let i = 1; i < k; i++) {
```

Comment:
Be careful with your types here. `k` is a python object. We shouldn't do this type of comparison between a python object and a javascript object.

Reply:
Ahh, yes. It's so easy to introduce a bug!